boards/silabs/radio_boards/slwrb4180b/board.cmake

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you confirm you are able to flash and debug the board even if this file is empty?